examples/js/postprocessing/DepthPass.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @author jjxtra / https://www.digitalruby.com
+ */
+
+THREE.DepthPass = function(scene, camera)
+{
+	THREE.Pass.call(this);
+
+	this.scene = scene;
+	this.camera = camera;
+	this.needsSwap = false;
+};
+
+THREE.DepthPass.prototype = Object.assign( Object.create( THREE.Pass.prototype ), {
+
+	constructor: THREE.DepthPass,
+
+	render: function ( renderer, writeBuffer, readBuffer /*, deltaTime, maskActive */ ) {
+
+		this.scene.overrideMaterial = null;
```

Comment:
Do you need to remember the current value and restore it at the end?

Reply:
I don't think so, RenderPass.js sets it to null at the end

Reply:
OK.